gooresmed/upload.py

```diff
@@ -417,14 +431,21 @@ def _process_response(self, response):
             # Tombstone the current upload so it cannot be used again.
             self._finished = True
         elif status_code == PERMANENT_REDIRECT:
-            bytes_range = _helpers.header_required(response, u'range')
+            try:
+                bytes_range = _helpers.header_required(response, u'range')
+            except exceptions.InvalidResponse:
```

Comment:
I "fixed it" by using a callback instead: https://github.com/GoogleCloudPlatform/google-resumable-media-python/commit/297387a12d9908ec194037e67120d0accafa4b55